### PR TITLE
Address missing author on analysis-runner by using audit_logs

### PR DESCRIPTION
**Pull request by illusional**
First more concrete application of audit_logs for analysis-runner log.

### CODE REVIEW

File: web/src/pages/project/AnalysisRunnerView/AnalysisRunnerSummary.tsx [hunk 1]
```diff
@@ -80,20 +83,24 @@ const AnalysisRunnerSummary: React.FunctionComponent = () => {
...
-        'Hail Batch': meta?.batch_url
-            ? meta?.batch_url.replace('https://batch.hail.populationgenomics.org.au/', '')
-            : '',
-        GitHub: `${meta?.repo}@${meta?.commit.substring(0, 7)}`,
-        Date: `${parseDate(sanitiseValue(meta?.timestamp))}`,
-        Author: `${parseEmail(sanitiseValue(author))}`,
-        Image: meta?.driverImage ? `${parseDriverImage(sanitiseValue(meta?.driverImage))}` : '',
-    }))
+    const flatData = data?.project.analyses.map(({ auditLogs, id, output, meta }, i) => {
+        const author = _.orderBy(auditLogs, ['timestamp'])[0]?.author
```
Comment (milo-hyben) [inline web/src/pages/project/AnalysisRunnerView/AnalysisRunnerSummary.tsx:87]: Should this have `orders='desc'` to display the latest author and not the first?

**Review by milo-hyben — APPROVED** [review]
Looking good, just I am not sure about the ordering of authors as per my comment.

File: web/src/pages/project/AnalysisRunnerView/AnalysisRunnerSummary.tsx (hunk 1 above)
Comment (illusional) [reply web/src/pages/project/AnalysisRunnerView/AnalysisRunnerSummary.tsx:87]: I want the first author, as that's the author that created the entry.

Comment (milo-hyben) [reply web/src/pages/project/AnalysisRunnerView/AnalysisRunnerSummary.tsx:87]: Make sense

**Pull request merged**